combat-trainer.lic

```diff
@@ -172,6 +172,12 @@ class SetupProcess
     else
       @equipment_manager.wield_weapon(game_state.weapon_name, game_state.weapon_skill)
     end
+
+    # Invoke Focus
+    if game_state.weapon_skill == 'Targeted Magic'
+      bput("invoke #{game_state.weapon_name}", 'You')
```

Comment:
So in the absence of an item, or maybe something like `Targeted Magic: ''` don't invoke the focus